### PR TITLE
Rebrand scenes patform folders v2

### PR DESCRIPTION
**Pull request by Norrox**
OK, so i firstly by mistake started to rebrand on the 4.3 stable fork instead of this one. 
I cherry picked my commit but it came with a few merge error witch i think i solved.

please check for errors and tell me if something needs changing / removed  / added 
i might have renamed something i should not have... 

It compiles fine on my linux machine
EDIT: And compiles to windows

### CODE REVIEW

**Review by Murly117 — APPROVED** [review]

File: platform/android/java/lib/patches/com.google.android.vending.expansion.downloader.patch [hunk 1]
```diff
@@ -97,7 +97,7 @@ index 979352299..3771d19c9 100644
      private static class Stub implements IStub {
          private IDownloaderService mItf = null;
 -        final Messenger mMessenger = new Messenger(new Handler() {
-+        // -- GODOT start --
++        // -- BLAZIUM start --
 +        private final MessengerHandlerServer mMsgHandler = new MessengerHandlerServer(this);
```
Comment (RandomOfNoWhere) [inline platform/android/java/lib/patches/com.google.android.vending.expansion.downloader.patch:101]: The line 101-103 maybe from a different branch, please validate this is required for rebranding

File: platform/android/java/lib/patches/com.google.android.vending.expansion.downloader.patch [hunk 2]
```diff
@@ -160,7 +160,7 @@ index 979352299..3771d19c9 100644
 +                    break;
```
Comment (RandomOfNoWhere) [inline platform/android/java/lib/patches/com.google.android.vending.expansion.downloader.patch:160]: These lines (160-162) maybe from a different branch, please validate this is required for rebranding

File: platform/android/java/lib/patches/com.google.android.vending.expansion.downloader.patch [hunk 3]
```diff
@@ -173,10 +173,10 @@ index e4b1b0f1c..36cd6aacf 100644
  import android.util.Log;
  
 -import com.android.vending.expansion.downloader.R;
-+// -- GODOT start --
++// -- BLAZIUM start --
 +//import com.android.vending.expansion.downloader.R;
```
Comment (RandomOfNoWhere) [inline platform/android/java/lib/patches/com.google.android.vending.expansion.downloader.patch:177]: The line 177-178 maybe from a different branch, please validate this is required for rebranding

File: platform/android/java/lib/patches/com.google.android.vending.expansion.downloader.patch [hunk 4]
```diff
@@ -185,15 +185,15 @@ index e4b1b0f1c..36cd6aacf 100644
              return "";
          }
 -        return String.format("%.2f",
-+        // -- GODOT start --
++        // -- BLAZIUM start --
 +        return String.format(Locale.ENGLISH, "%.2f",
```
Comment (RandomOfNoWhere) [inline platform/android/java/lib/patches/com.google.android.vending.expansion.downloader.patch:189]: This line maybe from a different branch, please validate this is required for rebranding

File: platform/android/java/lib/patches/com.google.android.vending.expansion.downloader.patch [hunk 5]
```diff
@@ -202,9 +202,9 @@ index e4b1b0f1c..36cd6aacf 100644
  
      public static String getSpeedString(float bytesPerMillisecond) {
 -        return String.format("%.2f", bytesPerMillisecond * 1000 / 1024);
-+        // -- GODOT start --
++        // -- BLAZIUM start --
 +        return String.format(Locale.ENGLISH, "%.2f", bytesPerMillisecond * 1000 / 1024);
```
Comment (RandomOfNoWhere) [inline platform/android/java/lib/patches/com.google.android.vending.expansion.downloader.patch:206]: The line 205 maybe from a different branch, please validate this is required for rebranding

File: platform/android/java/lib/patches/com.google.android.vending.expansion.downloader.patch [hunk 6]
```diff
@@ -266,12 +266,12 @@ index b2e0e7af0..c114b8a64 100644
              PowerManager pm = (PowerManager) mContext.getSystemService(Context.POWER_SERVICE);
 -            wakeLock = pm.newWakeLock(PowerManager.PARTIAL_WAKE_LOCK, Constants.TAG);
 -            wakeLock.acquire();
-+            // -- GODOT start --
++            // -- BLAZIUM start --
 +            //wakeLock = pm.newWakeLock(PowerManager.PARTIAL_WAKE_LOCK, Constants.TAG);
```
Comment (RandomOfNoWhere) [inline platform/android/java/lib/patches/com.google.android.vending.expansion.downloader.patch:270]: The line 270-273 maybe from a different branch, please validate they are required for rebranding

File: platform/android/java/lib/patches/com.google.android.vending.licensing.patch [hunk 7]
```diff
@@ -6,9 +6,9 @@ index 7c42bfc28..feb579af0 100644
      public void putString(String key, String value) {
          if (mEditor == null) {
              mEditor = mPreferences.edit();
-+            // -- GODOT start --
++            // -- BLAZIUM start --
 +            mEditor.apply();
```
Comment (RandomOfNoWhere) [inline platform/android/java/lib/patches/com.google.android.vending.licensing.patch:10]: This line maybe from a different branch, please validate they are required for rebranding

File: platform/android/java/lib/src/org/godotengine/godot/error/Error.kt
Comment (RandomOfNoWhere) [inline platform/android/java/lib/src/org/godotengine/godot/error/Error.kt:1]: This file maybe from a different branch, please validate this is required for rebranding

File: platform/android/java/lib/patches/com.google.android.vending.expansion.downloader.patch [hunk 8]
```diff
@@ -185,15 +185,15 @@ index e4b1b0f1c..36cd6aacf 100644
              return "";
          }
 -        return String.format("%.2f",
-+        // -- GODOT start --
++        // -- BLAZIUM start --
 +        return String.format(Locale.ENGLISH, "%.2f",
                  (float) overallProgress / (1024.0f * 1024.0f))
                  + "MB /" +
 -                String.format("%.2f", (float) overallTotal /
 +                String.format(Locale.ENGLISH, "%.2f", (float) overallTotal /
```
Comment (RandomOfNoWhere) [inline platform/android/java/lib/patches/com.google.android.vending.expansion.downloader.patch:193]: This line maybe from a different branch, please validate this is required for rebranding

File: platform/android/java/lib/patches/com.google.android.vending.expansion.downloader.patch [hunk 9]
```diff
@@ -216,9 +216,9 @@ index 12edd97ab..a0e1165cc 100644
  import android.telephony.TelephonyManager;
  import android.util.Log;
  
-+// -- GODOT start --
++// -- BLAZIUM start --
 +import android.annotation.SuppressLint;
```
Comment (RandomOfNoWhere) [inline platform/android/java/lib/patches/com.google.android.vending.expansion.downloader.patch:220]: This maybe from a different branch, please validate this is required for rebranding

File: platform/android/java/lib/patches/com.google.android.vending.expansion.downloader.patch [hunk 10]
```diff
@@ -248,10 +248,10 @@ index f1536e80e..4b214b22d 100644
  package com.google.android.vending.expansion.downloader.impl;
  
 -import com.android.vending.expansion.downloader.R;
-+// -- GODOT start --
++// -- BLAZIUM start --
 +//import com.android.vending.expansion.downloader.R;
```
Comment (RandomOfNoWhere) [inline platform/android/java/lib/patches/com.google.android.vending.expansion.downloader.patch:252]: The lines 251-252 maybe from a different branch, please validate this is required for rebranding

File: platform/android/java/lib/patches/com.google.android.vending.expansion.downloader.patch [hunk 11]
```diff
@@ -283,9 +283,9 @@ index 4babe476f..8d41a7690 100644
  import android.telephony.TelephonyManager;
  import android.util.Log;
  
-+// -- GODOT start --
++// -- BLAZIUM start --
 +import android.annotation.SuppressLint;
```
Comment (RandomOfNoWhere) [inline platform/android/java/lib/patches/com.google.android.vending.expansion.downloader.patch:287]: This line maybe from a different branch, please validate this is required for rebranding

File: platform/android/java/lib/patches/com.google.android.vending.licensing.patch [hunk 12]
```diff
@@ -32,11 +32,11 @@ import org.godotengine.godot.BuildConfig;
      }
  
 -    assert (e == outBuff.length);
-+    // -- GODOT start --
++    // -- BLAZIUM start --
 +    //assert (e == outBuff.length);
```
Comment (RandomOfNoWhere) [inline platform/android/java/lib/patches/com.google.android.vending.licensing.patch:36]: The lines 35-38 maybe from a different branch, please validate this is required for rebranding

**Review by RandomOfNoWhere — COMMENTED** [review]
Maybe a few lines that were left over from the Redot PR that are not related to rebranding. Please validate one way or the other :)

File: platform/android/java/lib/patches/com.google.android.vending.expansion.downloader.patch (hunk 1 above)
Comment (Norrox) [reply platform/android/java/lib/patches/com.google.android.vending.expansion.downloader.patch:101]: checked! this is same as the original

File: platform/android/java/lib/patches/com.google.android.vending.expansion.downloader.patch (hunk 2 above)
Comment (Norrox) [reply platform/android/java/lib/patches/com.google.android.vending.expansion.downloader.patch:160]: checked! ,the same!

File: platform/android/java/lib/patches/com.google.android.vending.expansion.downloader.patch (hunk 3 above)
Comment (Norrox) [reply platform/android/java/lib/patches/com.google.android.vending.expansion.downloader.patch:177]: checked, the same!, this is some sort of patch note i think

File: platform/android/java/lib/src/org/godotengine/godot/error/Error.kt
Comment (Norrox) [reply platform/android/java/lib/src/org/godotengine/godot/error/Error.kt:1]: removed

**Comment by Norrox** [comment]
i close this in favour of #12

**Pull request closed without merging**